src/tests/utils.test.ts

```diff
@@ -0,0 +1,13 @@
+import * as utils from "../utils";
+
+describe("Test isTransparent", () => {
+  it("should return true when color is rgb transparent", () => {
+    expect(utils.isTransparent("#ff00")).toEqual(true);
+    expect(utils.isTransparent("#fff00000")).toEqual(true);
+    expect(utils.isTransparent("transparent")).toEqual(true);
+  });
+
+  it("should return false when color is not transparent", () => {
+    expect(utils.isTransparent("#ced4da")).toEqual(false);
+  });
+});
```

Comment:
I'd rather have a UI test instead of a unit test. Unit test tells us nothing much here.

Reply:
There is already a UI test added and we can add more if needed but for utils, I feel we should also have unit tests since utils are pure js functions that can be used at multiple places, and if we later export it to the package `excalidraw-utils` the unit test helps.

Reply:
Alright, I skimmed the test titles, but it seems it's testing even the case where it shouldn't show. Good.

Reply:
We should have both, and I agree with @ad1992 we should add more unit tests for our utils.. not in this PR, but in general